semantic-conventions/src/tests/data/yaml/metrics.yaml

```diff
@@ -18,6 +20,8 @@ groups:
     brief: "Measures the size of foo."
     instrument: histogram
     unit: "{bars}"
+    note: >
```

Comment:
Now all the metrics have notes. We should also still test the case without a note.

Reply:
Good point. Fixed in [4f3fb3e](https://github.com/open-telemetry/build-tools/pull/177/commits/4f3fb3e8c148fe04e0a410b857dc5734893bd8ba)